examples/with-cookie-auth/utils/auth.js

```diff
@@ -11,18 +11,16 @@ export const login = ({ token }) => {
 export const auth = ctx => {
   const { token } = nextCookie(ctx)
 
-  /*
-   * If `ctx.req` is available it means we are on the server.
-   * Additionally if there's no token it means the user is not logged in.
-   */
-  if (ctx.req && !token) {
-    ctx.res.writeHead(302, { Location: '/login' })
-    ctx.res.end()
-  }
-
-  // We already checked for server. This should only happen on client.
+  // If there's no token, it means the user is not logged in.
   if (!token) {
-    Router.push('/login')
+    if (ctx.req) {
```

Comment:
```suggestion
    if (typeof window === 'undefined') {
```
This will allow webpack to remove the code that's not required in the build